### ISSUE TITLE
Unable to load ios app

### ISSUE THREAD

**Issue by kevinswartz** [BlinkID/blinkid-react-native#146]
Hi Guys,
I installed this plugin on Android without issue, but I'm having a hard time getting it to work on ios.  Here's the error I'm looking at:

```
dyld: Library not loaded: @rpath/Microblink.framework/Microblink
  Referenced from: /private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/<my_app_name>
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink: code signature invalid for '/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink'

	/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink: code signature invalid for '/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink'

	/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink: stat() failed with errno=1
	/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink: code signature invalid for '/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink'

	/private/var/containers/Bundle/Application/2139F27E-9218-43DD-9EB7-C6408F8CCB3B/<my_app_name>.app/Frameworks/Microblink.framework/Microblink: stat() failed with errno=1
```

Any ideas?  Thanks!
Kevin

**Comment by mparadina** [comment]
Hi @kevinswartz 

The issue could be linked with the .framework and .bundle files.

Can you please tell us which version of the SDK are you currently using?

Also, how are the .framework and .bundle named?
One more thing, what is the size of the .framework and .bundle files?

Regards,
Milan

**Comment by Cerovec** [comment]
Hi @kevinswartz,

Can you please check if this is helpful to you? https://stackoverflow.com/questions/32730312/reason-no-suitable-image-found

Also, are you able to run the sample app generated by `initReactNativeDemoApp.sh` script?

Best,
Jurica.

**Comment by kevinswartz** [comment]
Thanks for the help guys!

@mparadina
I'm on version 5.3.0.  The .framework and .bundle files haven't been renamed anywhere, they're default.  They're named Microblink.bundle, and Microblink.framework.  Microblink.bundle is 4.2Mb and Microblink.framework is 81.2Mb.

@Cerovec 
I went through the steps mentioned in the stackoverflow article, but nothing seemed to change anything for me.  I ran the demo project via `initReactNativeDemoApp.sh`, and ran into the same Xcode error.

**Comment by kevinswartz** [comment]
BTW, I should mention that I don't see the same thing if I run the demo app in the simulator.  If I do, the app boots, but then immediately fails with:
```
null is not an object (evaluating '_RNGestureHandlerModule.default.Direction')
```

**Comment by kevinswartz** [comment]
Hope this helps someone, but it sounds like there's an issue on ios 13.3.1 where deploying apps from a non-paid apple developer account is broken.  If I deploy to the simulator it works, if I deploy to an older iphone I have (running 12.2), it works.  It's supposedly fixed in ios 13.4, currently in beta.

Here's where I got this from:
https://github.com/Alamofire/Alamofire/issues/3051
